packages/node_modules/@webex/plugin-meetings/src/meeting/index.js

```diff
@@ -102,20 +102,20 @@ const logRequest = (request, {header = '', success = '', failure = ''}) => {
  */
 
 /**
-  * AudioVideo
-  * @typedef {Object} AudioVideo
-  * @property {Object} audio
-  * @property {String} audio.deviceId
-  * @property {Object} video
-  * @property {String} video.deviceId
-  */
+ * AudioVideo
```

Comment:
Is your editor auto adjusting everything? Can you make sure to only make changes related to the scope of work?

Reply:
I thought `npm` runs `eslint --fix`?  But I will revert and check.